spec/sec-regexp-regular-expression-objects-patch.html

```diff
@@ -356,71 +449,65 @@ <h1>%RegExpStringIteratorPrototype%.next ( )</h1>
-            1. <ins>Perform ? Set(_R_, *"lastIndex"*, _nextIndex_, *true*).</ins>
-            1. <ins>Set _lastIndex_ to _nextIndex_.</ins>
-          1. <ins>Else, set _lastIndex_ to _thisIndex_.</ins>
+          1. <ins>Let _matchValue_ be ? CoerceMatchResult( ? Get(_match_, "0"), _matchOptions_).</ins>
+          1. <del>If _matchStr_ is the empty String, then</del>
+          1. <ins>If ? IsEmptyMatch(_match_, _matchOptions_) is *true*, then</ins>
+            1. Let _thisIndex_ be ? ToLength(? Get(_R_, `"lastIndex"`)).
+            1. Let _nextIndex_ be AdvanceStringIndex(_S_, _thisIndex_, _fullUnicode_).
+            1. Perform ? Set(_R_, `"lastIndex"`, _nextIndex_, *true*).
           1. Return ! CreateIterResultObject(_match_, *false*).
         1. Else,
           1. Set _O_.[[Done]] to *true*.
           1. Return ! CreateIterResultObject(_match_, *false*).
       </emu-alg>
     </emu-clause>
-
-    <emu-clause id="sec-properties-of-regexp-string-iterator-instances">
-      <h1>Properties of RegExp String Iterator Instances</h1>
-      <p>RegExp String Iterator instances are ordinary objects that inherit properties from the <emu-xref href="#sec-%regexpstringiteratorprototype%-object">%RegExpStringIteratorPrototype%</emu-xref> intrinsic object. RegExp String Iterator instances are initially created with the internal slots listed in <a href="#table-1">Table 1</a>.</p>
-      <emu-table id="table-properties-of-regexp-string-iterator-instances" caption="Internal Slots of RegExp String Iterator Instances">
-        <table>
-          <tbody>
-            <tr>
-              <th>Internal Slot</th>
-              <th>Description</th>
-            </tr>
-            <tr>
-              <td>[[IteratingRegExp]]</td>
-              <td>The regular expression used for iteration. IsRegExp([[IteratingRegExp]]) is always initially *true*.</td>
-            </tr>
-            <tr>
-              <td>[[IteratedString]]</td>
-              <td>The String value being iterated upon.</td>
-            </tr>
-            <tr>
-              <td>[[Global]]</td>
-              <td>A Boolean value to indicate whether the [[IteratingRegExp]] is global or not.</td>
-            </tr>
-            <tr>
-              <td>[[Unicode]]</td>
-              <td>A Boolean value to indicate whether the [[IteratingRegExp]] is in Unicode mode or not.</td>
-            </tr>
-            <tr>
-              <td><ins>[[Mapper]]</ins></td>
-              <td><ins>Either *undefined*, or an ECMAScript function object used to map the offsets of a capture to a user-defined value.</ins></td>
-            </tr>
-            <tr>
-              <td>[[Done]]</td>
-              <td>A Boolean value to indicate whether the iteration is complete or not.</td>
-            </tr>
-            <tr>
-          </tbody>
-        </table>
-      </emu-table>
-    </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-properties-of-regexp-string-iterator-instances">
```

Comment:
should this still be nested under the `sec-%regexpstringiteratorprototype%-object` section?

Reply:
It isn't nested under that section in the ECMAScript spec, and shouldn't be here either as far as I can tell.